metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/MetadataImporterMainApplication.java

```diff
@@ -18,13 +18,72 @@
 
 package uk.ac.ebi.ampt2d.metadata.importer;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import uk.ac.ebi.ampt2d.metadata.importer.objectImporters.ObjectsImporter;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Study;
+import uk.ac.ebi.ampt2d.metadata.persistence.repositories.StudyRepository;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 @SpringBootApplication
-public class MetadataImporterMainApplication {
+public class MetadataImporterMainApplication implements ApplicationRunner {
```

Comment:
We already agreed in the past that the module would be renamed as `metadata-import` once the code was stabilized, because that is a lot more specific task than `load`.